pyworklistserver/worklist_server.py

```diff
@@ -44,11 +70,28 @@ def stop(self):
 
     def _on_find(self, event, app_logger):
         """ Event handler for C-FIND requests """
-        worklist_items = self._worklist_factory.get_worklist()
-        app_logger.info('Created worklist with {} exams'.format(len(worklist_items)))
-        for worklist_item in worklist_items:
+        seed = self.get_seedNumber()

```

Comment:
See random.getstate()/setstate(). I think that the client should decide the initial seed. Then, to the config file, you can write the state instead of the seed. 

When starting the server, you can then pass a config file (containing the state). If the caller does not also set the --seed, the server should read the state from the config file and set the state to the random number generator.